packages/components/src/box-control/utils.ts

```diff
@@ -180,6 +180,7 @@ export function isValuesMixed(
 export function isValuesDefined( values?: BoxControlValue ) {
 	return (
 		values !== undefined &&
+		values !== null &&
 		Object.values( values ).filter(
```

Comment:
This feels a bit verbose. I wonder if we can just do:

```suggestion
		values &&
		Object.values( values ).filter(
```

Although I'm fine with keeping it as you suggested.

Reply:
Certainly, this suggestion makes sense because values ​​other than `undefined` and `null` (`false`, `0`, `""`, `NaN`, etc.) can also be considered falsy 👍